types/src/sector/mod.rs

```diff
@@ -2,18 +2,16 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 pub mod post;
-mod registered_proof;
-mod seal;
-
 pub use self::post::*;
-pub use self::registered_proof::*;
-pub use self::seal::*;
 
-use crate::ActorID;
-use encoding::{repr::*, tuple::*};
+pub use fvm_shared::sector::{
+    AggregateSealVerifyInfo, AggregateSealVerifyProofAndInfos, InteractiveSealRandomness,
+    OnChainWindowPoStVerifyInfo, PoStProof, PoStRandomness, RegisteredAggregateProof,
+    RegisteredPoStProof, RegisteredSealProof, SealRandomness, SealVerifyInfo, SealVerifyParams,
+    SectorID, SectorInfo, SectorSize, WindowPoStVerifyInfo, WinningPoStVerifyInfo,
+};
 use num_bigint::BigInt;
-use num_derive::FromPrimitive;
-use std::fmt;
+pub const RANDOMNESS_LENGTH: usize = 32;
 
 /// SectorNumber is a numeric identifier for a sector. It is usually relative to a miner.
 pub type SectorNumber = u64;
```

Comment:
Should we also re-export `SectorNumber` and `SectorQuality` from fvm_shared even though they're just type aliases?

Reply:
and a bunch of others I didn't notice before, thanks!